cli/cmd/install.go

```diff
@@ -105,7 +105,7 @@ control plane.`,
   linkerd install | kubectl apply -f -
 
 The installation can be configured by using the --set, --values, --set-string and --set-file flags.
-A full list of configurable values can be found at https://www.github.com/linkerd/linkerd2/tree/main/charts/linkerd2/README.md`,
+A full list of configurable values can be found at https://www.github.com/linkerd/linkerd2/blob/main/charts/linkerd-control-plane/README.md`,
```

Comment:
While, this is definitely an improvement, it still seems fraught to have a link from released CLI versions to the `main` branch of the repo, for exactly the same reasons that caused this link to break previously.

It would be better to at least link to the published artifacts like https://artifacthub.io/packages/helm/linkerd2-edge/linkerd-control-plane; but even this is difficult because of the edge/stable distinction. Do we publish this information on linkerd.io's docs? Or perhaps we can refer to linkerd's docs more generally?

Reply:
What about linking to https://artifacthub.io/packages/helm/linkerd2/linkerd2#values ?

Reply:
@adleong That's the old chart being replaced in 2.12

Reply:
https://artifacthub.io/packages/helm/linkerd2/linkerd-control-plane#values then, with the expectation that the chart will be published during the release?